js/animations.js

```diff
@@ -0,0 +1,68 @@
+//Close the menu (shows up on md size) when clicked anywhere on the page
+$(document).on('click',function(){
```

Comment:
Spacing needs to be fixed. Use `eslint` to make sure you don't commit or PR with any spacing mistakes or just general errors.